package.json

```diff
@@ -71,6 +71,7 @@
     "@aws-sdk/client-s3": "3.588.0",
     "@aws-sdk/client-sqs": "3.588.0",
     "@slack/bolt": "3.18.0",
+    "is-valid-uuid-v4": "1.2.3",
```

Comment:
the code is so minimal and MIT licensed, let's not introduce a new dependency, but add a function to `spacecat-shared-utils`: https://github.com/jammymalina/is-valid-uuid-v4/blob/master/src/index.ts